azure-pipelines.yml

```diff
@@ -520,6 +579,28 @@ jobs:
       REDUCE_SDK: 'True'
       AZURE_CLI_TEST_TARGET_PROFILE: '2019-03-01'
 
+- job: TestExtensionsLoading
+  displayName: Test Extensions Loading
+  dependsOn: BuildPythonWheel
+  condition: succeeded()
+  timeoutInMinutes: 10
+
+  pool:
+    vmImage: 'ubuntu-16.04'
+  strategy:
+    matrix:
+      Python36:
+        python.version: '3.6'
+  steps:
+    - task: UsePythonVersion@0
+      displayName: 'Use Python $(python.version)'
+      inputs:
+        versionSpec: '$(python.version)'
+    - bash: pip install --upgrade pip wheel
```

Comment:
curious about this step. this is for test, does that mean user need to do this step when using cli? if yes, this should be in requirements.txt

Reply:
No need for user.
Just an extra step in script `./scripts/ci/test_extensions.sh` that build wheels and install them. So here package `wheel` must be installed ahead.

Actually, this step should be omitted in the future.

Reply:
It's actually requirements.txt